src/main/java/book/store/bookstore/repositories/BookRepository.java

```diff
@@ -0,0 +1,7 @@
+package book.store.bookstore.repositories;
+
+import book.store.bookstore.models.Book;
+import org.springframework.data.jpa.repository.JpaRepository;
+
+public interface BookRepository extends JpaRepository<Book,Long> {
```

Comment:
in this homework expected to write own implementation of BookRepository, e.g. using Hibernate
![image](https://github.com/k1llzers/book-store/assets/109764850/8f728eea-6be5-41b0-a3d6-d019fe813c68)